### ISSUE TITLE
Missing/Incorrect illustrations for pattern options

### ISSUE THREAD

**Issue by joostdecock** [freesewing/freesewing.org#190]
Below is a list of illustrations that are missing. Note that some illustrations are (currently) not available on the website, but we do have them (they just haven't been published yet). The ones below are ones that still need to be drawn:

## Missing illustrations

These are illustrations that have never been made. These should be priority.

 - [ ] **Bent**
    - [ ] Sleeve bend
    - [ ] Sleevecap ease
    - [ ] Sleeve length bonus

## Incorrect illustrations

These are illustrations that are shared between patterns, which means they are not strictly correct, but often are helpful nevertheless.
While it would be good to an illustration per option per pattern, these are somewhat less urgent as at least there's something.

 - [ ] **Bent**: All options except the ones listed above re-use illustrations from Brian

**Comment by ennis-s** [comment]
Here is my new account

**Comment by ennis-s** [comment]
![simon-01](https://user-images.githubusercontent.com/55888892/65786363-d9b0cf80-e199-11e9-9613-cdb9ed472b58.jpg)
I have to go to bed but if you could check this against the default shirt, I'd like to know if I've included all mentioned topstitching so far.

**Comment by joostdecock** [comment]
Hey wow, that's really great. Since you ask: There is normally no topstitching at the shoulder seam (from the shoulder to the neck) yet there is topstitching around the collar edge.

**Comment by ennis-s** [comment]
I read in the instructions that the yoke was edgestitched, which if I interpreted correctly shows on the top so would be shown in the illustration. Is the collar edgestitched or is it further back like on the cuffs?

**Comment by joostdecock** [comment]
The yoke is topstitched at the seam where it's joined to the back, not where it's joined to the front (which is what we see in the illustration).

The collar is topstiched further back, like the cuffs indeed.

**Comment by ennis-s** [comment]
ah I understand, got it
if you could send a clear photo of the cuff placket it would really help for when I do the back

**Comment by ennis-s** [comment]
![simon-01-01](https://user-images.githubusercontent.com/55888892/65813954-09ee8180-e21f-11e9-866c-e9544400b0fb.jpg)
Let me know if there's anything you can see that's wrong or could be adjusted for greater accuracy, then I'll upload it as an svg

**Comment by joostdecock** [comment]
Since you ask 😁

 - the back darts run vertical in the back, so while they do shape the garment, they always appear vertical in the end result (since top and bottom of the dart have the same amount of fabric between them)
 - the cuffs have small pleats since the sleeve is wider than the cuff

I don't think the cuff pleats are a big deal, but the back darts do throw me off 😉

**Comment by ennis-s** [comment]
No problem, easy fix

**Comment by ennis-s** [comment]
![simon-01](https://user-images.githubusercontent.com/55888892/65814222-5c7d6d00-e222-11e9-9e4c-9781a477e8a1.jpg)
Does the overall length look right? Should the bottom hem be slightly longer or do you think it's correct for the default?

**Comment by joostdecock** [comment]
That looks perfect. The length is good too 👏👏👏

**Comment by stoffsuchti** [comment]
@ennis-s very nice :) this will be good advertisment for the patterns on freesewing 😄👍

**Comment by joostdecock** [comment]
![newsimon](https://user-images.githubusercontent.com/1708494/66043450-77502a00-e51f-11e9-91f5-60c8f8bec31b.png)

I've adapted this for use in our linedrawing component. The reason it needed adaptation is because it uses filled paths, which change color in dark/light mode.

This is not required for the illustrations for the options, just for the main linedrawing

**Comment by ennis-s** [comment]
Can you send or link me that version? I just want to do some adjustments on the collar and it's easier than trying to describe it.

**Comment by joostdecock** [comment]
Yes, absolutely. It meant to do it but it slipped my mind. I've sent it via email just now

**Comment by ephphatha** [comment]
The [waralee instructions](https://en.freesewing.org/docs/patterns/waralee/instructions/) reference two images (waralee.svg and waralee-cutout.svg) which don't exist in that folder in the markdown repo. Could they be added to the list? I assume waralee-cutout is a isolated image of the cutout template but I'm not sure what the layout image (waralee.svg) was meant to contain.

**Comment by joostdecock** [comment]
I think this is a holdout from an older documentation template where we used to include:

 - The linedrawing (waralee-cutout.svg)
 - An example of the complete pattern (waralee.svg)
 
 We've moved linedrawings to their own React component, en we no longer include a pattern layout because it's not very helpful.

I'd have to look at it, but I suspect we can just drop these references.

**Comment by tangerineshark** [comment]
Hey would either of these variations work for the Hugo ribbing stretch pattern option (as-is or with corrections)? 
![ribbingstretch](https://user-images.githubusercontent.com/70777269/108021692-b41da380-6fe4-11eb-967a-5b5ec5b83f02.png)

**Comment by nataliabarnhart** [comment]
@tangerineshark I like the second one with the arrows!
